packages/api/schema/input.json

```diff
@@ -2,6 +2,66 @@
   "$ref": "#/definitions/ModelInput",
   "$schema": "http://json-schema.org/draft-07/schema#",
   "definitions": {
+    "Actual": {
+      "additionalProperties": false,
+      "properties": {
+        "cases": {
+          "type": "number"
+        },
+        "cumulativeCases": {
+          "type": "number"
+        },
+        "cumulativeDeaths": {
+          "type": "number"
+        },
+        "date": {
+          "$ref": "#/definitions/ISODate"
+        },
+        "deaths": {
+          "type": "number"
+        },
+        "mobility": {
+          "type": "number"
+        }
+      },
+      "required": [
+        "date",
+        "cases",
+        "cumulativeCases",
+        "deaths",
+        "cumulativeDeaths"
+      ],
+      "type": "object"
+    },
+    "HistoricalData": {
+      "additionalProperties": false,
+      "properties": {
+        "actuals": {
+          "items": {
+            "$ref": "#/definitions/Actual"
+          },
+          "type": "array"
+        },
+        "endDate": {
+          "$ref": "#/definitions/ISODate"
+        },
+        "totalCases": {
```

Comment:
How is `totalCases` different from `cumulativeCases`? Could we just assume that `endDate` is the latest date in the `actuals` array?